### PR TITLE
Move v2 subgraph endpoint to TempleDAO org

### PR DESCRIPTION
**Pull request by medariox**
# Description
What does this PR solve?

Fixes # (issue)

# Checklist
- [x] Code follows the style guide
- [x] I have performed a self-review of my own code
- [x] New and existing tests pass locally
- [x] This PR is targeting the correct branch

### CODE REVIEW

**Review by frontier159 — APPROVED** [review]

File: apps/automation-v2/src/config/prodnets.ts
Comment (frontier159) [inline apps/automation-v2/src/config/prodnets.ts:12]: @medariox was this ready to go and in sync now?

If so, @frost-ostrich could you redeploy the TLC liquidation bot with this latest once merged?

Comment (medariox) [reply apps/automation-v2/src/config/prodnets.ts:12]: Yep, it's in sync. Just waiting for frost-ostrich's changes to merge.

Comment (frost-ostrich) [reply apps/automation-v2/src/config/prodnets.ts:12]: Let's merge this, and then I'll redeploy the TLC liquidation bot, I don't need to do any src change here

**Pull request merged**
